app/controllers/mixins/compressed_ids.rb

```diff
@@ -1,6 +1,7 @@
 module CompressedIds
+  CID_REGEXP = '[0-9]r?[0-9]*'.freeze
```

Comment:
would prefer if we used compressed_id?  (regexp already defined for us), see lib/extensions/ar_region.rb